docs/public-networks/how-to/bonsai-limit-trie-logs.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Use Bonsai Tries to reduce database size
+sidebar_position: 12
+description: Enable this feature to reduce the size of your database
+tags:
+  - public networks
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Reduce database size
+
+You can use the the early access feature `--Xbonsai-limit-trie-logs-enabled` to minimize the database size of Besu, if you are using `data-storage-format=BONSAI`. When enabled, this feature can reduce database growth by more than 3GB each week on mainnet.
+
+To use this feature before it is enabled by default, add the `--Xbonsai-limit-trie-logs-enabled` option to your Besu command.
+
+## Limit Trie Logs for Bonsai
+
+:::caution
+
+The following commands are examples. Before executing these example commands on your node, m,modify them to apply to your node's configuration.
+
+:::
+
+1. Add the `--Xbonsai-limit-trie-logs-enabled` option to the [Besu configuration file](configuration-file).
+1. Stop Besu.
+1. (optional) Run the following command to run the Besu client for the Bonsai data storage format, specifying a data directory for pruned trie logs:
```

Comment:
Suggest: "Run the Besu client and specify the Bonsai Trie data storage format and a data directory for the pruned trie logs:"

Then add a fenced code block for the command